tests/e2e/e2e_exec_test.go

```diff
@@ -405,110 +303,30 @@ func (s *IntegrationTestSuite) executeGaiaTxCommand(ctx context.Context, c *chai
-		time.Second,
-		"tx returned a non-zero code; stdout: %s, stderr: %s", outBuf.String(), errBuf.String(),
+		15*time.Second,
+		2*time.Second,
 	)
-	endpoint = fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+
+	endpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
 	s.Require().Eventually(
 		func() bool {
 			return queryGaiaTx(endpoint, txResp.TxHash) == nil
 		},
 		time.Minute,
 		5*time.Second,
-		"stdout: %s, stderr: %s", outBuf.String(), errBuf.String(),
-	)
-}
-
-func (s *IntegrationTestSuite) queryGovProposal(endpoint string, proposalId uint64) (govv1beta1.QueryProposalResponse, error) {
-	var emptyProp govv1beta1.QueryProposalResponse
-
-	path := fmt.Sprintf("%s/cosmos/gov/v1beta1/proposals/%d", endpoint, proposalId)
-	resp, err := http.Get(path)
-	if err != nil {
-		s.T().Logf("This is the err: %s", err.Error())
-	}
-
-	if err != nil {
-		return emptyProp, fmt.Errorf("failed to execute HTTP request: %w", err)
-	}
-	defer resp.Body.Close()
-
-	body, err := io.ReadAll(resp.Body)
-	if err != nil {
-		s.T().Logf("This is the err: %s", err.Error())
-	}
-	if err != nil {
-		return emptyProp, err
-	}
-	var govProposalResp govv1beta1.QueryProposalResponse
-
-	if err := cdc.UnmarshalJSON(body, &govProposalResp); err != nil {
-		return emptyProp, err
-	}
-	s.T().Logf("This is the gov response: %s", govProposalResp)
-
-	return govProposalResp, nil
-}
-
-func (s *IntegrationTestSuite) getLatestBlockHeight(c *chain, valIdx int) int {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-	defer cancel()
-
-	type status struct {
-		LatestHeight string `json:"latest_block_height"`
-	}
-
-	type syncInfo struct {
-		SyncInfo status `json:"SyncInfo"`
-	}
-
-	var (
-		outBuf        bytes.Buffer
-		errBuf        bytes.Buffer
-		block         syncInfo
-		currentHeight int
-	)
-
-	s.Require().Eventually(
-		func() bool {
-			exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
-				Context:      ctx,
-				AttachStdout: true,
-				AttachStderr: true,
-				Container:    s.valResources[c.id][valIdx].Container.ID,
-				User:         "nonroot",
-				Cmd:          []string{"gaiad", "status"},
-			})
-			s.Require().NoError(err)
-
-			err = s.dkrPool.Client.StartExec(exec.ID, docker.StartExecOptions{
-				Context:      ctx,
-				Detach:       false,
-				OutputStream: &outBuf,
-				ErrorStream:  &errBuf,
-			})
-			s.Require().NoError(err)
-			s.Require().NoError(json.Unmarshal(errBuf.Bytes(), &block))
-
-			currentHeight, err = strconv.Atoi(block.SyncInfo.LatestHeight)
-			s.Require().NoError(err)
-			return currentHeight > 0
-		},
-		5*time.Second,
-		time.Second,
-		"Get node status returned a non-zero code; stdout: %s, stderr: %s", outBuf.String(), errBuf.String(),
 	)
-	return currentHeight
 }
 
-func (s *IntegrationTestSuite) execCreateGroup(c *chain, valIdx int, endpoint string, adminAddr string, metadata string, groupMembersPath string, fees string) {
```

Comment:
i think that gofumpt was adding explicit types to parameters instead of letting them be grouped like this.
we only started using gofumpt recently though and i'm unsure whether it's run as part of our CI or not at this point.
I guess we'd see an error on this PR if it had any problems with this.

all to say that I don't have a strong preference on either (a weak preference for the way you've done it hear), but would defer to our CI should we need to.